faraday/deviceconfiguration.py

```diff
@@ -230,7 +236,9 @@ def configureDeviceConfiguration(args, deviceConfigurationConfigPath, faradayCon
     # Create configuration boot bitmask integer
     bootmask = [0] * 8
     redledtx = fconfig.get('BASIC', 'REDLEDTX')
+    greenledrx = fconfig.get('BASIC', 'GREENLEDRX')
     unitconfigured = fconfig.get('BASIC', 'UNITCONFIGURED')
+    bootmask[2] = greenledrx
```

Comment:
Correct, I was confused by `bootbitmask.reverse()`. Thanks for clarifying, edit made.